### DISCUSSION TITLE
when v1.1

### DISCUSSION THREAD

**Discussion by Leather128** [rayanmargham/FNF-Godot-Engine#33] [Q&A] [answered]
when version 1.1 out plz need know now totally

**Comment by rayanmargham** [comment, marked as answer]
im currently busy with other projects im really sorry that not a lot of progress is happening I'll say probs early 2021 version 1.5 or 2.0

**Reply by Leather128** [reply]
2022 right? cuz it's already late 2021 lol

**Reply by rayanmargham** [reply]
lol yes early 2022

**Reply by mamrofy** [reply]
oh nice lol
